content/concepts/asset-manager.md

```diff
@@ -18,7 +18,7 @@ The Asset Manager is an Entreprise only feature, meaning all the following resou
 :::
 
 ## Asset family
-::: availability versions=3.2,4.0,Serenity editions=EE
+::: availability versions=3.2,4.x,5.x,Serenity editions=EE
```

Comment:
Maybe we should keep 4.0 instead of 4.x since there is no other version than 4.0

Reply:
Yes, you're right. I'll update the 4.x and 5.x with 4.0 and 5.0